### DISCUSSION TITLE
Markdown tables don't respect the table style set in reference.docx

### DISCUSSION THREAD

**Discussion by aburkov** [jgm/pandoc#10346] [General]
Here's my markdown:

| | <span custom-style="Verbatim Char">num_blocks</span> | <span custom-style="Verbatim Char">emb_dim</span> | <span custom-style="Verbatim Char">num_heads</span> | <span custom-style="Verbatim Char">vocab_size</span> |
|---------|---:|---:|---:|---:|
| Our model | 2 | 128 | 8 | 32,011 |
| Llama 3.1 8B | 32 | 4,096 | 32 | 128,000 |
| Llama 3.1 70B | 80 | 8,192 | 64 | 128,000 |
| Llama 3.1 405B | 126 | 16,384 | 128 | 128,000 |
| Gemma 2 9B | 126 | 3,584 | 16 | 256,128 |
| Gemma 2 27B | 42 | 4,608 | 32 | 256,128 |

Here's how it looks like in reference.docx:

<img width="104" alt="Screenshot 2024-11-02 at 4 43 38 PM" src="https://github.com/user-attachments/assets/c09be136-824d-43f7-a37c-e198e0aa267c">
<img width="784" alt="Screenshot 2024-11-02 at 4 45 07 PM" src="https://github.com/user-attachments/assets/23504a8a-55a0-4376-94e4-e6a842f68f3f">

And this is the rendered output:

<img width="508" alt="Screenshot 2024-11-02 at 4 44 46 PM" src="https://github.com/user-attachments/assets/b2702245-44ae-4efc-8e4c-426b1ec8dbff">

Did anyone experienced this problem? How to solve it?

**Comment by iandol** [comment]
Are you editing the "Table" style explicitly in your reference doc? It is a bit fiddly to select it and Word doesn't identify which style is being used for tables in the same way it does for text, but it is possible.

<img width="453" alt="image" src="https://github.com/user-attachments/assets/c15631d9-71c9-470f-be75-a53f3e4abda9">

**Reply by aburkov** [reply]
That did it, thanks a lot!
